### DISCUSSION TITLE
Theme Fluent. Assignments problem

### DISCUSSION THREAD

**Discussion by 100fotosdelviaje** [gantry/gantry5#3235] [Q&A]
When i click in "Assignments" nothing happend. The page remain "thinking"  (sorry about my inglish)  <img width="840" alt="Captura de pantalla 2024-03-03 10 32 15" src="https://github.com/gantry/gantry5/assets/162031842/d12e2a7d-3043-44db-a11c-bd728b92675f">

**Comment by N8Solutions** [comment]
@100fotosdelviaje The discussion and issues for the GitHub repo here is for Gantry 5 and the free Hydrogen & Helium themes. Any theme specific questions should be asked on the relevant forum. You didn't say what CMS you are using, so I'm providing you with the link for Joomla support on the Rocket Theme website.
https://rockettheme.com/forum/joomla-template-fluent
Good Luck!
